src/control/run_go_tests.sh

```diff
@@ -20,15 +20,15 @@ function find_build_source()
 
 function check_environment()
 {
-	if [ -z "$LD_LIBRARY_PATH" ]; then
+	if [ -z "${LD_LIBRARY_PATH:-}" ]; then
```

Comment:
Am I missing the subtlety of the empty `${...:-}` syntax here?

```bash
$ [ -z "$foo" ] && echo empty
empty
$ [ -z "${foo:-}" ] && echo empty
empty
```

Reply:
The first one will fail with `set -u` (which is enabled now).